app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
+
+> Why should I care?
+
+You could use help
+
+> What are the reasons I should care about getting more contributors for my open source project?
+
+- **Reduce your workload** - Regular contributors help
+- **Grow your skills and help others** - Whatever your skill level, working on open source software is a great way to learn skills like leadership, mentoring, and project management.
+- **Build your reputation** -
+- **Create better software** - people with different skills and passions working together create software that
+
+> How should I get started?
+
+## Join the community on Open Source Friday
+
+> Logistics: remove any barriers to making it happen this Friday
+
+### Set aside time
+
+TODO: words about what might block you (time, boss, etc)
+
+CTA: Add to calendar
+
+> Knowledge: What if I feel like I don't know enough yet?
+
+## Learn how to build a welcoming community
```

Comment:
Might be better to link to maintainer best practices (feels like less pressure than "build a community")